src/emulator/gui/src/game_selector.cpp

```diff
@@ -179,22 +190,105 @@ void draw_game_selector(GuiState &gui, HostState &host) {
+                                ImGui::LogFinish();
+                            }
+                            if (ImGui::MenuItem("ID")) {
+                                ImGui::LogToClipboard();
+                                ImGui::LogText("%s", game.title_id.c_str());
+                                ImGui::LogFinish();
+                            }
+                            if (ImGui::MenuItem("Name")) {
+                                ImGui::LogToClipboard();
+                                ImGui::LogText("%s", game.title.c_str());
+                                ImGui::LogFinish();
+                            }
+                            ImGui::EndMenu();
+                        }
+                        if (ImGui::BeginMenu("Remove")) {
+                            if (ImGui::MenuItem("Game"))
+                                gui.delete_game_popup = true;
+                            if (ImGui::MenuItem("Shader Log")) {
+                                if (fs::exists(shaderlog_path / game.title_id))
+                                    fs::remove_all(shaderlog_path / game.title_id);
+                            }
+                            ImGui::EndMenu();
+                        }
+                        ImGui::EndPopup();
+                    }
+
+                    if (gui.delete_game_popup)
+                        ImGui::OpenPopup("Delete Game");
+                    ImGui::PushStyleColor(ImGuiCol_Text, GUI_COLOR_TEXT_MENUBAR_OPTIONS);
+                    if (ImGui::BeginPopupModal("Delete Game", NULL, ImGuiWindowFlags_AlwaysAutoResize)) {
+                        ImGui::PopStyleColor();
+                        ImGui::Text("Do you really want delete this game ?\n%s [%s]", game.title_id.c_str(), game.title.c_str());
+                        if (ImGui::IsItemHovered())
+                            ImGui::SetTooltip("Deleting a game may take a few moments.\nDepending on the size of the games and your computer.");
+                        ImGui::Separator();
+                        static const ImVec2 CONTEXT_BUTTON_SIZE = ImVec2(124, 0);
+                        if (ImGui::Button("Yes", CONTEXT_BUTTON_SIZE)) {
+                            fs::remove_all(game_path / game.title_id);
+                            if (!fs::exists(game_path / gui.title_id)) {
+                                LOG_INFO("The game '{} [{}]' has been successfully deleted.", game.title_id.c_str(), game.title.c_str());
+                                gui.game_selector.games.erase(games_index);
+                                if (gui.user_backgrounds[host.cfg.background_image] && gui.current_background != static_cast<std::uint32_t>(gui.user_backgrounds[host.cfg.background_image]))
+                                    gui.current_background = gui.user_backgrounds[host.cfg.background_image];
+                                else if (!gui.user_backgrounds[host.cfg.background_image] && gui.current_background != 0)
```

Comment:
I needbthis check i have set this firstbtime for good raison i no want it set value if is already set is juste stupid so it juste check is background used but i can change this for more clear because now i using same code 2 tome

Reply:
I gave you the reasons T_T

Please make a change.